### ISSUE TITLE
Player Minimized After Exiting Fullscreen

### ISSUE THREAD

**Issue by akano12** [libre-tube/LibreTube#3468]
### Steps to reproduce

1. Open LT
2. Search for something
3. Play a video
4. Go to fullscreen
5. Exit fullscreen

### Expected behavior

Exit to portrait mode

### Actual behavior

Player is minimized 

### LibreTube version

0.13.1

### Android version

Android 13

### Other details

This only happens for the first video played after searching for something (first as in the first video you play not first in order).

So if you search for something and play a video then go to fullscreen, player will be minimized after exiting fullscreen. Videos you play next won't be affected unless you search foe something else.
tl;dr bug affects first video played in each search query

(crappy internet atm will upload recording if needed in the morning) 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by wizhel** [comment]
I have same problem, but for me Libretube crashed when i exit fullscreen

**Comment by wizhel** [comment]
```java
java.lang.NullPointerException
	at com.github.libretube.ui.fragments.PlayerFragment.unsetFullscreen(PlayerFragment.kt:3)
	at com.github.libretube.ui.fragments.PlayerFragment.onConfigurationChanged(PlayerFragment.kt:55)
	at androidx.fragment.app.FragmentManager.dispatchConfigurationChanged(FragmentManager.java:46)
	at androidx.fragment.app.FragmentManager$$ExternalSyntheticLambda0.accept(R8$$SyntheticClass:12)
	at androidx.activity.ComponentActivity.onConfigurationChanged(ComponentActivity.java:22)
	at androidx.appcompat.app.AppCompatActivity.onConfigurationChanged(AppCompatActivity.java:1)
	at com.github.libretube.ui.activities.MainActivity.onConfigurationChanged(MainActivity.kt:6)
	at android.app.ActivityThread.performActivityConfigurationChanged(ActivityThread.java:6185)
	at android.app.ActivityThread.performConfigurationChangedForActivity(ActivityThread.java:6041)
	at android.app.ActivityThread.performConfigurationChangedForActivity(ActivityThread.java:6019)
	at android.app.ActivityThread.handleActivityConfigurationChanged(ActivityThread.java:6544)
	at android.app.ActivityThread$ActivityClientRecord.lambda$init$0$ActivityThread$ActivityClientRecord(ActivityThread.java:783)
	at android.app.-$$Lambda$ActivityThread$ActivityClientRecord$HOrG1qglSjSUHSjKBn2rXtX0gGg.onConfigurationChanged(Unknown Source:2)
	at android.view.ViewRootImpl.performConfigurationChange(ViewRootImpl.java:5190)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:3003)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:2241)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:8793)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1031)
	at android.view.Choreographer.doCallbacks(Choreographer.java:849)
	at android.view.Choreographer.doFrame(Choreographer.java:779)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1016)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:257)
	at android.app.ActivityThread.main(ActivityThread.java:8335)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:626)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1032)
    ```

**Comment by akano12** [comment]
> I have same problem, but for me Libretube crashed when i exit fullscreen

U sure you have the latest version installed?

**Comment by wizhel** [comment]
Yes

**Comment by Bnyro** [comment]
Could you please share a screen recording? I can't really reproduce it, probably because I didn't get the issue properly.

**Comment by akano12** [comment]
> Could you please share a screen recording? I can't really reproduce it, probably because I didn't get the issue properly.

For some reason this issue isn't reproducible once a percentage of the video duration is crossed.

So far I've noticed that when videos are near the end, LT doesn't exhibit the same behavior.

So tl;dr:

1- Search for something -> play a video -> tap back btn -> issue reproducible

2- Search for something -> play a video -> set the time to over 90% of the duration -> issue isn't reproducible.

**Comment by akano12** [comment]
Only reproducible when back button is tapped when in full screen. 
Exiting fullscreen with buttom right button works just fine

**Comment by Bnyro** [comment]
MediaSessionCompat did only affect the notification, but no UI.

**Comment by akano12** [comment]
Can't test anything right now, piped instances aren't working and disabling piped proxy does nothing

**Comment by Bnyro** [comment]
hostux.net, adminforge.de and frontendfriendly.xyz have been working when I checked that the last time. Can't reproduce the disable proxy thing, works fine as well for me.

**Comment by Bnyro** [comment]
https://piped-instances.kavin.rocks/ seems to be down, not sure why :/

**Comment by akano12** [comment]
This issue doesn't exist in videos played from trending tab, only from search results

**Comment by Bnyro** [comment]
it's rather Piped that struggles now, really unfortunate that G is acting so angry

**Comment by akano12** [comment]
> ah yes.video from searches are still minimized,who's G and why he is angry,i mean LT is nothing but alter ego of Piped

uncle google is being mad at Aurora and Piped. Sad times for degooglers

**Comment by akano12** [comment]
Can't seem to reproduce this issue anymore on the debug build.

**Comment by Bnyro** [comment]
Neat, thanks for testing further :+1:

**Comment by akano12** [comment]
> Neat, thanks for testing further :+1:

What i noticed tho is that the screen goes blank while exiting fullscreen, was this a recent change (like transition animation?)
![20231105_173813.jpg](https://github.com/libre-tube/LibreTube/assets/50032603/0f59f772-9b87-4b58-a8f3-17d811af14e2)

**Comment by Bnyro** [comment]
> What i noticed tho is that the screen goes blank while exiting fullscreen, was this a recent change (like transition animation?)
>
Yes, that's probably because I've been re-writing the fullscreen logic in a different way to allow us to add a landscape layout for the player page. I'll see if I can find a way to fix that before the next release, but no promises (I think it's rather low priority for now).
